x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -114,11 +114,12 @@
                             "monitor", "manage_index_templates", MonitoringBulkAction.NAME, "manage_saml", "manage_token"
                         },
                         new RoleDescriptor.IndicesPrivileges[] {
-                                RoleDescriptor.IndicesPrivileges.builder().indices(".kibana*", ".reporting-*").privileges("all").build(),
+                                RoleDescriptor.IndicesPrivileges.builder()
+                                        .indices(".kibana*", ".reporting-*", ".code-*").privileges("all").build(),
```

Comment:
@spacedragon I assume this role is used for non Code admin user, who will only have READ permission to code indices. And to allow a user to have READ-ONLY permission, the admin of Kibana has to grant `kibana_user` role to the user, right? Here the `kibana_user` will have `all` permission to `.code-` indices though.

Reply:
The `kibana_user` role is slowly becoming more of a "Kibana administrator". Users with the `kibana_user` role have full read/write access to all spaces, so I think it makes sense in this context to have full access to the code indices. I don't know a whole lot about Code, so let me know if that sounds wrong.

Reply:
@legrego Yea, this makes sense. In our case, we probably need to have a role with only read permission as a *regular* Code user as opposed to an *admin* Code user.

Reply:
changed `code_system` to `code_admin`